Card.java

```diff
@@ -77,8 +77,6 @@ public boolean setAll(int value, char suit) {
             this.value = value;
             return true;
         }
-        value = 5;
-        suit = CLUB;

```

Comment:
The result should be the card 5 CLUB  and return false if the input values are not valid.